docs/grpc/java.md

```diff
@@ -30,13 +30,13 @@ with lnd in Java. We'll be using Maven as our build tool.
 Note the ***proto*** folder, where all the proto files are kept.
 
  - [rpc.proto](https://github.com/lightningnetwork/lnd/blob/master/lnrpc/rpc.proto)
- - [annotations.proto](https://github.com/grpc-ecosystem/grpc-gateway/blob/master/third_party/googleapis/google/api/annotations.proto)
- - [http.proto](https://github.com/grpc-ecosystem/grpc-gateway/blob/master/third_party/googleapis/google/api/http.proto)
+ - [annotations.proto](https://github.com/googleapis/googleapis/blob/master/google/api/annotations.proto)
```

Comment:
The `annotations.proto` and `http.proto` files aren't needed anymore since they're not included from the `rpc.proto` any longer. It would make sense to add lnd's `walletunlocker.proto` here though!